### PR TITLE
Cubzh Gigax Unit (Unit 3)

### PR DESCRIPTION
**Pull request by simoninithomas**
Context: Collaboration between Gigax x Cubzh x Hugging Face.

You can see how it looks here 👉 https://moon-ci-docs.huggingface.co/docs/ml-games-course/pr_22/en/unit3/introduction

### CODE REVIEW

File: units/en/unit3/customize.mdx [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# Customize your own demo
+
```
Comment (pcuenca) [inline units/en/unit3/customize.mdx:2]: Would also be cool to have a call-for-action for users to upload their creations as demo Spaces.

**Review by pcuenca — COMMENTED** [review]
Feel free to ping me for review when the content is ready :)

File: units/en/unit3/customize.mdx (hunk 1 above)
Comment (simoninithomas) [reply units/en/unit3/customize.mdx:72]: They will, to do the demo they need to duplicate the space

File: units/en/unit3/conclusion.mdx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+We hope you enjoyed exploring NPC-Playground and experiencing the future of gaming with smart LLM-powered NPCs. 
+
+Don't forget to:
+
+- Share your demo on LinkedIn and X, and tag us  @cubzh_ @gigax @huggingface **we'll repost it** 🤗. 
+
+- But also share it on the [Hugging Face Discord server in the #ml-4-games-i-made-this channel](https://hf.co/join/discord).
+
+ADD Illustration
```
Comment (pcuenca) [inline units/en/unit3/conclusion.mdx:13]: reminder: missing

File: units/en/unit3/customize.mdx [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+- Share your demo on LinkedIn and X, and tag us  @cubzh_ @gigax @huggingface **we'll repost it** 🤗. 
+
+- But also share it on the [Hugging Face Discord server in the #ml-4-games-i-made-this channel](https://hf.co/join/discord).
+
+## Teach your NPC new skills
+
+For this part of the unit, it's [Tristan](https://x.com/tr_deborde), the founder of Gigax, that made a video tutorial where you're going to teach a NPC new skills 🔥.
+
+**The name of the Space changed, it's not ai-npc anymore but NAME.**
```
Comment (pcuenca) [inline units/en/unit3/customize.mdx:19]:
```suggestion
**The name of the Space changed, it's not ai-npc anymore but NPC-Playground.**
```

I guess

File: units/en/unit3/customize.mdx [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+<video controls width="1280" height="720">
+  <source src="https://huggingface.co/datasets/huggingface-ml-4-games-course/course-images/resolve/main/en/unit3/gigax_cubzh_hf_full.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
+
+
+## Bonus: find a funny hat
+
+In addition, to being able to teach your NPCs new skills you can customize the assets you want to use in your demo thanks to a free library of 25k assets.
```
Comment (pcuenca) [inline units/en/unit3/customize.mdx:29]:
```suggestion
In addition to being able to teach your NPCs new skills, you can customize the assets you want to use in your demo thanks to a free library of 25k assets.
```

File: units/en/unit3/customize.mdx [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+You can replace it with one of the 6 hats we provided.
+
+Now, you just need to save, go back to your demo and talk to an NPC, ask them to put you a hat in your head and voila!
+
+
+<img src="https://huggingface.co/datasets/huggingface-ml-4-games-course/course-images/resolve/main/en/unit3/partyhat3.png" alt="Cubzh Hats" />
+
+
+
+## Challenge: create a new skills
```
Comment (pcuenca) [inline units/en/unit3/customize.mdx:67]:
```suggestion
## Challenge: create new skills
```

File: units/en/unit3/customize.mdx [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+## Teach your NPC new skills
+
+For this part of the unit, it's [Tristan](https://x.com/tr_deborde), the founder of Gigax, that made a video tutorial where you're going to teach a NPC new skills 🔥.
+
+**The name of the Space changed, it's not ai-npc anymore but NAME.**
+
+<video controls width="1280" height="720">
+  <source src="https://huggingface.co/datasets/huggingface-ml-4-games-course/course-images/resolve/main/en/unit3/gigax_cubzh_hf_full.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+</video>
```
Comment (mishig25) [inline units/en/unit3/customize.mdx:24]:
```suggestion
<video controls width="1280" height="720" src="https://huggingface.co/datasets/huggingface-ml-4-games-course/course-images/resolve/main/en/unit3/gigax_cubzh_hf_full.mp4">
</video>
```

File: units/en/unit3/demo.mdx [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+It's easier to think about this in terms of input/output flow:
+
+- In **input**, the model reads [a text description](https://github.com/GigaxGames/gigax/blob/main/gigax/prompt.py) of a 3D scene, alongside a description of the recent events and a list of the NPC's available actions (e.g., `<say>`, `<jump>`, `<attack>`, etc.).
+
+- The model then **outputs** one of these actions using parameters that refer to 3D entities that exist in the scene, e.g. `say NPC1 "Hello, Captain!"`.
+
+<img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/blog/181_npc-gigax-cubzh/gigax.png" alt="gigax" />
+
+Gigax has **open-sourced their stack!** You can download their [fine-tuned models on the 🤗 Hub](https://huggingface.co/Gigax), and clone their [inference stack on Github](https://github.com/GigaxGames/gigax).
```
Comment (tristandeborde) [inline units/en/unit3/demo.mdx:48]:
```suggestion
Gigax has **open-sourced their stack!** 
You can clone their [inference stack on Github](https://github.com/GigaxGames/gigax).
For this demo, their models are hosted in the cloud, but you can [download them yourself on the 🤗 Hub](https://huggingface.co/Gigax):
- [Phi-3 fine-tuned model](https://huggingface.co/Gigax/NPC-LLM-3_8B)
- [Mistral-7B fine-tuned model](https://huggingface.co/Gigax/NPC-LLM-7B)
```

File: units/en/unit3/customize.mdx [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+
+- But also share it on the [Hugging Face Discord server in the #ml-4-games-i-made-this channel](https://hf.co/join/discord).
+
+## Teach your NPC new skills
+
+For this part of the unit, it's [Tristan](https://x.com/tr_deborde), the founder of Gigax, that made a video tutorial where you're going to teach a NPC new skills 🔥.
+
+**The name of the Space changed, it's not ai-npc anymore but NAME.**
+
+<video controls width="1280" height="720" src="https://huggingface.co/datasets/huggingface-ml-4-games-course/course-images/resolve/main/en/unit3/gigax_cubzh_hf_full.mp4">
```
Comment (mishig25) [inline units/en/unit3/customize.mdx:21]:
```suggestion
<video width="1280" height="720" controls="true" src="https://huggingface.co/datasets/huggingface-ml-4-games-course/course-images/resolve/main/en/unit3/gigax_cubzh_hf_full.mp4">
```

**Review by tristandeborde — COMMENTED** [review]
Added some more details on the Gigax description

**Pull request merged**